[07] Word Scramble/Word Scramble/Models/WSGame.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  WSGame.swift
+//  Word Scramble
+//
+//  Created by Joe Pham on 2022-07-25.
+//
+
+import Foundation
+
+struct WSGame {
+	
+	var rootWord: String
+	var userAnswer: String = ""
+	var usedWords: [String] = []
+	var userScore: Int {
+		var score = 0
+		for usedWord in usedWords { score += usedWord.count }
+		return score
+	}
+	
+	private(set) var errorTitle = ""
+	private(set) var errorMessage = ""
+	var showingError = false
```

Comment:
Consider this could be in a separate struct/class. But this is good enough I guess, with 1 level of passing the "showingError" variable 👌